onnx/test/test_external_data.py

```diff
@@ -738,5 +740,16 @@ def test_check_model_absolute(self) -> None:
             checker.check_model(self.model_filename)
 
 
+class TestSaveAllTensorsAsExternalDataWithPath(TestSaveAllTensorsAsExternalData):
+    def get_temp_model_filename(self) -> pathlib.Path:
+        return pathlib.Path(super().get_temp_model_filename())
+
+
+class TestExternalDataToArrayWithPath(TestExternalDataToArray):
+    @property
+    def model_file_path(self):
```

Comment:
```suggestion
    def model_file_path(self) -> pathlib.Path:
```

Reply:
In the parent class, doing this would break the MyPy linting. Would it really be necessary to specify output types only in the child class? This feels wonky.

In case where parent class property is

```
@property
def model_file_path(self) -> str:
    return self._model_file_path
```

MyPy emits

```
>>> Lint for onnx/test/test_external_data.py:

  Error (MYPY) override
    Signature of "model_file_path" incompatible with supertype
    "TestExternalDataToArray"
    
    To disable, use `  # type: ignore[override]`

        747  |
        748  |class TestExternalDataToArrayWithPath(TestExternalDataToArray):
        749  |    @property
    >>> 750  |    def model_file_path(self) -> pathlib.Path:
        751  |        return pathlib.Path(self._model_file_path)
        752  |
        753  |
```